pkg/internal/checkup/vmi.go

```diff
@@ -53,15 +53,23 @@ const (
 	terminationGracePeriodSeconds = 0
 )
 
-func newVMIUnderTest(name string, checkupConfig config.Config) *kvcorev1.VirtualMachineInstance {
+func newVMIUnderTest(name string, checkupConfig config.Config, configMapName string) *kvcorev1.VirtualMachineInstance {
+	const (
+		configDiskSerial = "DEADBEEF"
```

Comment:
Please consider increasing the scope of the original `configDiskSerial` const, and share it between the VMIs.

Reply:
I don't see a reason to do that, It would imply that it is connected somehow.. 
Both VMIs have configmaps, both happen to have the same serial..